GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -493,7 +493,7 @@ private bool TryExtractUrl(string contents, out string url)
             var parts = contents.Split(' ');
             foreach (string s in parts)
             {
-                if (Uri.IsWellFormedUriString(s, UriKind.Absolute))
+                if (PathUtil.CanBeGitURL(s))
```

Comment:
This will also include Git directories, which clone cannot handle.

Reply:
Fair, but the current logic doesn't even pull valid SSH URLs used by one of the biggest (if not the biggest) centralised git solutions in the world.

This is essentially a revert without losing the new functionality of pulling the git URL from a paragraph of text.

Reply:
Not arguing bout that, just that `CanBeGitURL` is probably to general. 
If git-clone accepts Git directories, this is OK but the comment should be changed

Reply:
You can clone from a bare repository on a file system directory.
Which comment are you referring to?

Reply:
Line 476

Reply:
Got it, thanks.  I've updated.